modules/nodes-pods-user-namespaces-configuring.adoc

```diff
@@ -97,43 +80,49 @@ The range 1000/10000 means 10,000 values starting with ID 1000, so it specifies
 apiVersion: v1
 kind: Pod
 metadata:
+  namespace: userns
   name: userns-pod
-
 # ...
-
 spec:
-  containers:
-  - name: userns-container
-    image: registry.access.redhat.com/ubi9
-    command: ["sleep", "1000"]
-    securityContext:
-      capabilities:
-        drop: ["ALL"]
-      allowPrivilegeEscalation: false <1>
-      runAsNonRoot: true <2>
-      seccompProfile:
-        type: RuntimeDefault
-      runAsUser: 1000 <3>
-      runAsGroup: 1000 <4>
-  hostUsers: false <5>
-
+#...
+  template:
+    metadata:
+      labels:
+        app: name
+      annotations:
+        openshift.io/required-scc: "restricted-v3" <1>     
+    spec:
+      hostUsers: false <2>
+      containers:
+      - name: userns-container
+        image: registry.access.redhat.com/ubi9
+        command: ["sleep", "1000"]
+        securityContext:
+          capabilities:
+            drop: ["ALL"]
+          allowPrivilegeEscalation: false
+          runAsNonRoot: true <3>
+          procMount: Unmasked <4>
+          runAsUser: 1000 <5>
+          runAsGroup: 1000 <6>
 # ...
 ----
-<1> Specifies that a pod cannot request privilege escalation. This is required for the `restricted-v2` security context constraints (SCC).
-<2> Specifies that the container will run with a user with any UID other than 0.
-<3> Specifies the UID the container is run with.
-<4> Specifies which primary GID the containers is run with.
-<5> Requests that the pod is to be run in a user namespace. If `true`, the pod runs in the host user namespace. If `false`, the pod runs in a new user namespace that is created for the pod. The default is `true`.
+<1> Specifies the SCC to use with this workload.
+<2> Requests that the pod is to be run in a user namespace. If `true`, the pod runs in the host user namespace. If `false`, the pod runs in a new user namespace that is created for the pod. The default is `true`.
+<3> Specifies that the container will run with a user that has any UID other than 0.
+<4> Optional: Specifies the type of proc mount to use for the containers. The `unmasked` value ensures that a container's `/proc` file system is mounted as read/write by the container process. The `unmasked` parameter should be used only with an SCC that sets `hostUsers` to `false`. The default is `Default`.
+<5> Specifies the UID of the container. This must fall in the range that you set in the namespace object. 
+<6> Specifies the GID of the containers. This must fall in the range that you set in the namespace object.
```

Comment:
same as above: this is in reference to inside the container, and outside will be arbitrarily high and randomly chosen by kubelet